doc/ref_impl/cntt-ri/chapters/chapter08.md

```diff
@@ -447,44 +447,44 @@ skipping tests if optional services (e.g. Gnocchi, Barbican, etc.) or
+| opnfv/functest-smoke:hunter | rally_sanity      | Success            |
+| opnfv/functest-smoke:hunter | refstack_defcore  | Success            |
+| opnfv/functest-smoke:hunter | tempest_full      | Success            |
+| opnfv/functest-smoke:hunter | tempest_scenario  | Success            |
+| opnfv/functest-smoke:hunter | tempest_slow      | Success            |
+| opnfv/functest-smoke:hunter | patrole           | Success            |
+| opnfv/functest-smoke:hunter | neutron_trunk     | Success            |
+| opnfv/functest-smoke:hunter | networking-bgpvpn | Success            |
+| opnfv/functest-smoke:hunter | networking-sfc    | Success            |
+| opnfv/functest-smoke:hunter | barbican          | Success            |
 
 According to [RA1 Core OpenStack Services APIs]({{ "/doc/ref_arch/openstack/chapters/chapter05.html" | relative_url }})
 and [RC1 TC Requirements]({{ "doc/ref_cert/lfn/chapters/chapter09.html" | relative_url }})
 the following test cases are executed at the end.
 
-| container                               | test case                  | criteria (Jenkins) |
-|-----------------------------------------|----------------------------|:------------------:|
-| opnfv/functest-smoke-cntt:hunter        | neutron-tempest-plugin-api | Success            |
-| opnfv/functest-smoke-cntt:hunter        | tempest_cinder             | Success            |
-| opnfv/functest-smoke-cntt:hunter        | tempest_keystone           | Success            |
-| opnfv/functest-smoke-cntt:hunter        | rally_sanity               | Success            |
-| opnfv/functest-smoke-cntt:hunter        | tempest_full               | Success            |
-| opnfv/functest-smoke-cntt:hunter        | tempest_scenario           | Success            |
-| opnfv/functest-smoke-cntt:hunter        | tempest_slow               | Success            |
-| opnfv/functest-benchmarking-cntt:hunter | rally_full                 | Success            |
-| opnfv/functest-benchmarking-cntt:hunter | rally_jobs                 | Success            |
-| opnfv/functest-benchmarking-cntt:hunter | vmtp                       | Success            |
-| opnfv/functest-benchmarking-cntt:hunter | shaker                     | Success            |
-| opnfv/functest-vnf:hunter               | cloudify                   | Success            |
-| opnfv/functest-vnf:hunter               | cloudify_ims               | Success            |
-| opnfv/functest-vnf:hunter               | heat_ims                   | Success            |
-| opnfv/functest-vnf:hunter               | vyos_vrouter               | Success            |
-| opnfv/functest-vnf:hunter               | juju_epc                   | Success            |
+| container                               | test case        | criteria (Jenkins) |
+|-----------------------------------------|------------------|:------------------:|
+| opnfv/functest-smoke-cntt:hunter        | tempest_neutron  | Success            |
```

Comment:
should the test cases not be tempest-neutron-cntt, tempest-cinder-cntt, ... for the first seven cases?

Reply:
You're right. I should have changed them in my previous change.
Fixing here. Thank you!